### ISSUE TITLE
Compostion data analysis

### ISSUE THREAD

**Issue by gfrc11752** [Huh/PopR_SDGFP#7]
In the plots tab could we produce plots that compare F:D ratios across years. Year on the x-axis, F:D on the y, to determine significant differences within units across years? I am noticing a lot of incomplete data sets in the composition tab. Missing years and groups compared to what is in the Cybertracker database.

**Comment by Huh** [comment]
This is a little trickier.  The IPM only cares about a single year estimate and that is why the website is setup to work that way.  I did have this setup to do multiple years previously though and I could implement that, but it will take some effort.  Why don't we leave this issue open and I will implement it when we have more time.

**Comment by Huh** [comment]
I have been thinking about this and because the time series of the ratios is an output of the IPM I would rather inference were made from the IPM where the disparate sources of data must agree.  

Can I close this given the IPM argument or do you still want people to look at the time-series and mentally synthesize the sources of data?....not being snarky it is the way I think about how we make inference from multiple sources of the data.

**Comment by gfrc11752** [comment]
I agree with the IPM argument; however, I know I am going to get a request to look at how recruitment changed through time even though that will be reflected in the IPM. Lets keep it on the radar, and if time prevails and its not to difficult try to throw in a lumped herd comp graph through whatever years are selected. Not a priority though...
